packages/react/runtime/src/debug/profile.ts

```diff
@@ -2,51 +2,44 @@
 // Licensed under the Apache License Version 2.0 that can be found in the
 // LICENSE file in the root directory of this source tree.
 import { options } from 'preact';
-import type { Component, ComponentClass, VNode } from 'preact';
+import type { ComponentClass, VNode } from 'preact';
 
 import { COMPONENT, DIFF, DIFFED, RENDER } from '../renderToOpcodes/constants.js';
-import { getDisplayName } from '../utils.js';
+import { getDisplayName, hook } from '../utils.js';
 
 export function initProfileHook(): void {
-  const oldDiff = options[DIFF];
-  options[DIFF] = function(vnode: VNode) {
-    // This __PROFILE__ is used for DCE testing
-    if (__PROFILE__ && typeof vnode.type === 'function') {
+  const profileStart = lynx.performance?.profileStart;
```

Comment:
Do we need to fallback to `console.profile` when `lynx.performance?.profileStart === undefined`?